static/home.js

```diff
@@ -8,7 +8,15 @@
     async function getRandomGame() {
         clearRandomGameResponseElements();
         const user = document.getElementById("user").value;
-        const response = await fetch('/random_game?user=' + user)
+        const minplayers = document.getElementById("number-of-players-min").value;
+        const maxplayers = document.getElementById("number-of-players-max").value;
+        const playerrangetype = "normal"
+        if(document.getElementById("normal-range-players").checked){
+            const playerrangetype = "normal"
+        }else if(document.getElementById("exact-range-players").checked) {
+            const playerrangetype = "exact"
+        }
+        const response = await fetch(`/random_game?user=${user}&minplayers=${minplayers}&maxplayers=${maxplayers}&playerrangetype=${playerrangetype}`)
```

Comment:
[TO DO BEFORE MERGE] Need to account for if minplayers and/or maxplayers are not set in the front end they are set to None.  Currently this causes a 500 error Failed to load resource `http://127.0.0.1:5000/random_game?user=JDGiardino&minplayers=&maxplayers=&playerrangetype=normal`
<img width="548" alt="Screen Shot 2022-09-07 at 9 23 02 AM" src="https://user-images.githubusercontent.com/14614633/188889653-1810c201-13bf-4861-830c-41947ffb078a.png">

Reply:
Done!